code/solutions/package-and-deploy/scripts/invoke-lambda-function.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# From code/package-and-deploy directory run ./scripts/invoke-lambda-function.sh
+
+LAMBDA_FUNCTION_NAME="cfn-workshop-python-function"
+
+PAYLOAD='{"time_zone":"Europe/London"}'
```

Comment:
we should probably let this be user defined given it is a user specific settings, then provide a default

Reply:
Done